SalesforceFoundation/EDA
# prettier plugin apex and update prettierrc.yml

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes
to run prettier:
1. Make sure you have `yarn` installed.
2. Run `yarn install` to install development dependencies
3. Run `yarn prettier --check .` for warning/error msgs of or `yarn prettier --write .` to autoformatting all the code.
